drivers/python/README.md

```diff
@@ -1,3 +1,46 @@
 # Python Driver
 
 For more information about this driver see the [Python API](http://www.rethinkdb.com/api/python/) page.
+
+# Contributing
+
+## Installing
+If you would like to start helping develop the python driver you need to do
+the following:
+
+```bash
+git clone https://github.com/rethinkdb/rethinkdb.git
+cd rethinkdb
+./configure --allow-fetch
+make
+cd drivers/python
+make
+pip install -e .
+```
+
+Since the python driver requires you to run the `make` command for the whole
```

Comment:
And for completeness, you can run testing using an existing install using the `--server-binary` option. Not sure how that is going to play here.